File: result/translate/transcoder/java2python/python/ELEMENTS_TO_BE_ADDED_SO_THAT_ALL_ELEMENTS_OF_A_RANGE_ARE_PRESENT_IN_ARRAY_1.py
def f_gold ( arr , n ) :
    s = set ( )
    count , maxm , minm = 0 , int ( min ( arr ) ) , int ( max ( arr ) )
    for i in range ( n ) :
        s.add ( arr [ i ] )
        if arr [ i ] < minm :
            minm = arr [ i ]
        if arr [ i ] > maxm :
            maxm = arr [ i ]
    for i in range ( minm , maxm + 1 ) :
        if not s.add ( i ) :
            count += 1
    return count

File: result/translate/transcoder/python2java/java/ELEMENTS_TO_BE_ADDED_SO_THAT_ALL_ELEMENTS_OF_A_RANGE_ARE_PRESENT_IN_ARRAY_1.java
public static int f_gold ( int [ ] arr , int n ) {
  Map s = new HashMap ( ) ;
  int count = 0 , maxm = - 10 * 9 , minm = 10 * 9 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    s . put ( arr [ i ] , 1 ) ;
    if ( ( arr [ i ] < minm ) && ( arr [ i ] > maxm ) ) minm = arr [ i ] ;
    if ( ( arr [ i ] > maxm ) && ( arr [ i ] < minm ) ) maxm = arr [ i ] ;
  }
  for ( int i = minm ;
  i <= maxm ;
  i ++ ) if ( i != 0 ) count ++ ;
  return count ;
}
